### PR TITLE
Change style

### PR DESCRIPTION
**Pull request by ricardovogel**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,47 +3,40 @@
...
+        </div>
+
+        <h5>Privacy voor prominente studenten.</h5>
+
+        <img src="images/pizza.png" onclick="pizzaTime()" class="pizza" id="pizza"
+          alt="Ter vervanging van hopeloos verouderde cookies, maakt DuckDuckVo gebruik van pizza\'s.">
+      </div>
+    </main>
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
```
Comment (svenvdvoort) [inline index.html]: Where is this dependency used? If it's not used, we can remove it.
```suggestion
```

**Review by svenvdvoort — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (ricardovogel) [reply index.html]: This is a script that comes with Bootstrap. I'll check if it's necessary in our case.

Comment (ricardovogel) [reply index.html]: Deleting the line doesn't change the functionality, I'll remove it and the others.

Comment (svenvdvoort) [reply index.html]: Great, thanks!

**Review by svenvdvoort — APPROVED** [review]

**Pull request merged**